src/main/scala/nl.knaw.dans.easy.deposit/docs/dm/Date.scala

```diff
@@ -42,7 +42,26 @@ case class Date(
                  scheme: Option[String],
                  value: Option[String],
                  qualifier: Option[DateQualifier],
-               )
+               ) {
+
+  /**
+   * If the scheme is present and is equal to DateScheme.W3CDTF and
+   * if the value is present and starts with a yyyy-MM-dd formatted date
+   * converts this Date with date-and-time value to just day level.
+   * Else, keep the value as is
+   *
+   * @return the day level representation of this Date
+   */
+  def toDayLevel: Date = {
+    lazy val hasW3CDTFScheme = scheme contains DateScheme.W3CDTF.toString
+    lazy val hasValidValue = value.exists(_ matches "^[0-9]{4}-[0-9]{2}-[0-9]{2}(.*)$")
+
+    if (hasW3CDTFScheme && hasValidValue)
+      copy(value = value.map(DateTime.parse(_).toString(ISODateTimeFormat.date())))
```

Comment:
I don't prefer this, because it also gets rid of the validation that this is a 'real date' and the proper formatting of a date (in case of single digit months/days I prefer them to be displayed as double digits).